### PR TITLE
Update CONTRIBUTING.md with info about signing off from the github ui

### PR DESCRIPTION
**Pull request by susanev**
Signed-off-by: susanev <susan.ra.evans@gmail.com>

## Description
added a sentence about how to sign off when using the GitHub web ui to make changes

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for the [Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ The DCO requires a sign-off message in the following format appear on each commi
 Signed-off-by: Julia Child <juliachild@chef.io>
 ```
 
-The DCO text can either be manually added to your commit body, or you can add either **-s** or **--signoff** to your usual git commit commands. If you forget to add the sign-off you can also amend a previous commit with the sign-off by running **git commit --amend -s**. If you've pushed your changes to GitHub already you'll need to force push your branch after this with **git push -f**.
+The DCO text can either be manually added to your commit body, or you can add either **-s** or **--signoff** to your usual git commit commands. If you are using the GitHub UI to make a change you can add the sign-off message directory to the pull request description. If you forget to add the sign-off you can also amend a previous commit with the sign-off by running **git commit --amend -s**. If you've pushed your changes to GitHub already you'll need to force push your branch after this with **git push -f**.
```
Comment (msorens) [inline CONTRIBUTING.md]: s/directory/directly/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ The DCO requires a sign-off message in the following format appear on each commi
...
 
 ### Chef Obvious Fix Policy
```
Comment (msorens) [inline CONTRIBUTING.md]: In this following section, the "obvious fix" policy discusses an exemption to the DCO requirement for small things... but as far as I know that is not true for the `automate` repository: if you do not have a DCO the PR will fail its requisite checks.

Comment (tas50) [reply CONTRIBUTING.md]: Many things do not require a DCO signoff. https://github.com/chef/chef/blob/master/CONTRIBUTING.md#chef-obvious-fix-policy

Comment (tas50) [reply CONTRIBUTING.md]: This was the policy that legal drafted. Expeditor is simply enforcing sign-offs on a PR. What's written here is what we should be enforcing.

**Comment by tas50** [comment]
@msorens The contributing documentation is presented when beginning the process of opening a PR against a new repository. Additionally, you'll get a DCO check against your PR which will link you to the DCO documentation.

**Pull request merged**